src/walletConnect/screens/DappsDisclaimer.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text } from 'react-native'
+import { useSelector } from 'react-redux'
+import { dappConnectInfoSelector, dappsMinimalDisclaimerEnabledSelector } from 'src/dapps/selectors'
+import { DappConnectInfo } from 'src/dapps/types'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  isDappListed: boolean
+}
+
+const DappsDisclaimer = ({ isDappListed }: Props) => {
+  const { t } = useTranslation()
+  const dappsMinimalDisclaimerEnabled = useSelector(dappsMinimalDisclaimerEnabledSelector)
+  const dappConnectInfo = useSelector(dappConnectInfoSelector)
+
+  if (dappsMinimalDisclaimerEnabled) {
+    return (
+      <Text style={styles.dappNotListedDisclaimer}>
+        {isDappListed ? t('dappsDisclaimerSingleDapp') : t('dappsDisclaimerUnlistedDapp')}
+      </Text>
+    )
+  }
+
+  if (!isDappListed && dappConnectInfo === DappConnectInfo.Basic) {
```

Comment:
i know this is not a 1:1 change as before, we used to also show this disclaimer on the dapp connect bottom sheet but i think the direction we want to move to is to display disclaimers on "risky" actions only, and we had discussed how connecting to a dapp is very low risk